dev/doc/release-process.md

```diff
@@ -86,4 +89,3 @@
   + The MacOS packages should be signed with our own certificate.
   + The Snap package has to be built and uploaded to the snap store by running a [platform CI job manually](https://github.com/coq/platform/tree/2021.02/linux/snap/github_actions).
     Then ask a Platform maintainer with access to the store to publish the upload or give you access so that you can do it yourself.
-- [ ] Announce the release to Coq-Club and Discourse (coq-club@inria.fr + coq+announcements@discoursemail.com).
```

Comment:
Probably the whole section "## This is now delegated to the platform maintainers" should be removed, because it is clearly not maintained by the people involved and is becoming seriously outdated.